samples/Qrom/QromSample.qs

```diff
@@ -1,4 +1,4 @@
-﻿namespace implicitQRAM {
+﻿namespace QromSample {
```

Comment:
This file has ImplicitQRAM some places and QROM in others - but let's take care of this when we get to the QROMs.